pubspec.yaml

```diff
@@ -1,17 +1,17 @@
 name: dynamic_config_generator
 description: Dynamically generate config for a build variant using a json file
-version: 0.2.0
```

Comment:
@ahujamanish Can we bump the version to 0.2.1 instead of 0.3.0 as it's just dependency updates?